frontend/src/container/QueryBuilder/components/DataSourceDropdown/DataSourceDropdown.tsx

```diff
@@ -10,7 +10,9 @@ import { QueryLabelProps } from './DataSourceDropdown.interfaces';
 
 const dataSourceMap = [DataSource.LOGS, DataSource.METRICS, DataSource.TRACES];
 
-export function DataSourceDropdown(props: QueryLabelProps): JSX.Element {
+export const DataSourceDropdown = memo(function DataSourceDropdown(
```

Comment:
let's use memo while exporting the default component not here

Reply:
Is it cause some problems in naming export?

Reply:
no there is no issue in named export memorisation


but here we are trying to follow one pattern in the project to use default export only which would be consistent across different component